runtime/vm/createramclass.cpp

```diff
@@ -1782,11 +1785,13 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 					if (interfaceClass == NULL) {
 						return FALSE;
 					}
+					#if JAVA_SPEC_VERSION < 24
 					if (requirePackageAccessCheck(vm, classLoader, module, interfaceClass)
```

Comment:
The local `vm` was removed: this must use `vmThread->javaVM` (like lin 1730) or the local must return.